.github/workflows/main.yml

```
@@ -48,14 +48,14 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: 16
-      - run: yarn --ignore-scripts
+      - run: yarn # Need to build to get rust bindings
```

```suggestion
      - run: yarn # Need to build scripts to get rust bindings
```

Just to make it more obvious :)